src/server.js

```diff
@@ -43,7 +43,9 @@ import router from './router';
 import chunks from './chunk-manifest.json'; // eslint-disable-line import/no-unresolved
 import config from './config';
 
-require('dotenv').config();
+const dotenvConfig = require('dotenv').config();
+console.log('XXX JMF dotenvConfig', { dotenvConfig });
```

Comment:
This line prints the expected object:

```
XXX JMF dotenvConfig {
  dotenvConfig: {
    parsed: {
      GEO_APP_KEY: 'CENSORED'
      ...
    }
  }
}
```

Reply:
fixed in https://github.com/josephfrazier/reported-web/pull/427/commits/2837fe8baca5e32a419d6c0c00e41c2d7aa74981